core/modules/node/config/views.view.content_recent.json

```
@@ -0,0 +1,561 @@
+{
+    "_config_name": "views.view.content_recent",
+    "name": "content_recent",
+    "description": "Recent content.",
```

Let's retain the description of the legacy block (which should also be translatable):

```suggestion
    "_config_name": "views.view.content_recent",
    "_config_translatables": [
        "description"
    ],
    "name": "content_recent",
    "description": "A list of recently published content.",
```